Celeste.Mod.mm/Mod/Registry/DecalRegistry.cs

```diff
@@ -156,11 +156,13 @@ public static class DecalRegistry {
         /// <summary>
         /// Reads a DecalRegistry.xml file's contents
         /// </summary>
-        public static void ReadDecalRegistryXml(string fileContents) {
+        public static List<KeyValuePair<string, DecalInfo>> ReadDecalRegistryXml(string fileContents) {
```

Comment:
Personally not a huge fan of changing the method behaviors and signatures in general, as mods could be depending on this method already. In this specific case though, it makes sense, and I'd be willing to hotglue around this with some MonoModLinkFrom if a mod ever crashes from this API change 😅